tests/unit/records/serializers/literature/common/test_reference_item.py

```diff
@@ -107,3 +107,69 @@ def test_returns_non_empty_fields_if_some_fields_missing():
     result = schema.dumps(dump).data
 
     assert expected == json.loads(result)
+
+
+def test_returns_no_misc_if_title_persent():
+    schema = ReferenceItemSchemaV1()
+    dump = {
+        'reference': {
+            'title': {
+                'title': 'Jessica Jones',
+            },
+            'misc': [
+                'A Misc',
+            ]
+        }
+    }
+    expected = {
+        'titles': [
+            {
+                'title': 'Jessica Jones'
+            }
+        ]
+    }
+    result = schema.dumps(dump).data
+
+    assert expected == json.loads(result)
+
+
+def test_returns_no_misc_if_titles_persent():
+    schema = ReferenceItemSchemaV1()
```

Comment:
is the ```titles`` inside the ``reference`` key?

Reply:
it might be good to add schema validation (like e.g. here:https://github.com/inspirehep/inspire-next/blob/773de8e374bd154bb12927527e106b51b0a5f87a/tests/unit/workflows/test_workflows_tasks_actions.py#L78-L110) to all serializer tests, so you know whether the test cases are valid, and you also know what you have to change when the schema changes as the tests will break.